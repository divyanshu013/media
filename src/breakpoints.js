const breakpoints = {
  xsmall: 420,
  small: 576,
  medium: 768,
  large: 992,
  xlarge: 1200,
};

export default breakpoints;
