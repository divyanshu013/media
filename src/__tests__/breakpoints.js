import breakpoints from '../breakpoints';

test('breakpoints snapshot', () => {
  expect(breakpoints).toMatchSnapshot();
});
