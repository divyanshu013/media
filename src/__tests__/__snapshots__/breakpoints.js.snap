// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpoints snapshot 1`] = `
Object {
  "large": 992,
  "medium": 768,
  "small": 576,
  "xlarge": 1200,
  "xsmall": 420,
}
`;
